microprofile-fault-tolerance/README.adoc

```diff
@@ -41,14 +41,33 @@ include::../shared-doc/use-of-jboss-home-name.adoc[leveloffset=+1]
 The application built in this guide simulates a simple backend for a gourmet coffee on-line store. It implements a REST
 endpoint providing information about coffee samples we have in store.
 
-Let's imagine, although it's not implemented as such, that some of the methods in our endpoint require communication
+Let's imagine, although it's not implemented as such, that some methods in our endpoint require communication
 to external services like a database or an external microservice, which introduces a factor of unreliability.
 
 == Solution
 
 We recommend that you follow the instructions in the next sections and create the application step by step.
 However, you can go right to the completed example.
 
+// System Requirements
+include::../shared-doc/system-requirements.adoc[leveloffset=+1]
+
+// Use of {jbossHomeName}
+include::../shared-doc/use-of-jboss-home-name.adoc[leveloffset=+1]
```

Comment:
this is already included above

Reply:
Thanks, The section needs to be shuffled - fixing now.

Reply:
Fixed – Prerequisites section is now after solution section.